lib/matplotlib/axis.py

```diff
@@ -1825,8 +1825,13 @@ def set_ticklabels(self, ticklabels, *, minor=False, **kwargs):
             For each tick, includes ``tick.label1`` if it is visible, then
             ``tick.label2`` if it is visible, in that order.
         """
-        ticklabels = [t.get_text() if hasattr(t, 'get_text') else t
-                      for t in ticklabels]
+        ticklabels = []
```

Comment:
This line is breaking the other tests because it is overriding the passed in `ticklabels`.